src/safari-app-extension/AlpheiosSafari/BackgroundProcess.swift

```diff
@@ -119,6 +122,8 @@ class BackgroundProcess {
             if (curTab.isActive) {
                 self.activateContent(tab: curTab, window: window)
             } else {
+                curTab.setPanelDefault() // Reset panel state to a default
```

Comment:
I found that we do because the way Safari works it will send whatever state is has stored on some occasions (don't remember exactly what it was, discovered it during testing). So that might be our best (and maybe even only) place to reset the state. Basically, deactivation resets state, so we can do it here, on my opinion. Later we might not have enough information to distinguish between situations where we need to send a stored state to the content script or if we need to reset it first.